### DISCUSSION TITLE
Task 3 - Assignment 2

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#17] [Week 1 - Questions] [answered]
I'm working through the Github starter course with the Optional Next Steps at the bottom of the page. I believe I opened a pull request to let the teacher know I finished the course (first bullet point), but I'm struggling to know how to create a markdown (second bullet point). I'm not sure which repository to post in, and I'm not sure exactly what a markdown is yet. I've read the link to the page about markdowns, but it doesn't define it clearly. Is markdown similar to HTML coding? Should my post be in the pull request I made or posted elsewhere?

**Comment by kassyoveson** [comment, marked as answer]
I went back to the assignment page to go through the tasks. I struggled with this because I named my markdown wrong and it didn't check me off, so I had to do it twice. I think I made my own repository, but I could have made it on the class link. I am really hoping this class gets easier. I have really struggled this week.

**Reply by metasean** [reply]
The actual difficulty will definitely be easier.  The amount of work is still going to be a challenge.  And there are some students who have a had time transferring what they've learned to their project, but my goal is that you will know how to ask for (and give) assistance when you need it.  You should have resources in the class-repo, and you can ask questions here, the UtahJS slack channel, and office hours.  If you need some additional assistance outside of office hours, just ping me and I'll see if we can find a time that works with both our schedules for a zoom.
